index.css

```diff
@@ -442,7 +644,7 @@ li {
 
   #contact-me {
     background-color: #6070ff;
-    padding: 120px 20px 30px 20px;
+    padding: 20px 20px;
     background-image: url("./images/mobile-contact-bg.png");
     background-size: 100%;
     overflow: hidden;
```

Comment:
- [ ] There's overlapping of background image on this contact-me section. Kindly look into it and fix.

###  Your Design
![your-contact-me-left-logo](https://user-images.githubusercontent.com/26597432/220656327-d0593e5a-e298-4560-b856-6ac34f907374.png)

### Figma Design
![temp-contact-me-left-logo](https://user-images.githubusercontent.com/26597432/220656419-54c2d583-201f-4ea3-a4d9-c8799010909c.png)